geode-assembly/src/main/dist/bin/gfsh.bat

```diff
@@ -74,14 +70,14 @@ REM )
 REM  Consider java is from JDK
 @set TOOLS_JAR=%JAVA_HOME%\lib\tools.jar
 @IF EXIST "%TOOLS_JAR%" (
-    @set CLASSPATH=%CLASSPATH%;%TOOLS_JAR%
+    @set DEPENDENCIES=%DEPENDENCIES%;%TOOLS_JAR%
 ) ELSE (
     set TOOLS_JAR=
 )
 
 @set LAUNCHER=com.gemstone.gemfire.management.internal.cli.Launcher
 @if defined JAVA_ARGS (
-@set JAVA_ARGS="%JAVA_ARGS%"
+    @set JAVA_ARGS="%JAVA_ARGS%"
 )
-@"%GF_JAVA%" -Dgfsh=true -Dlog4j.configurationFile=classpath:log4j2-cli.xml %JAVA_ARGS% %LAUNCHER% %*

```

Comment:
I am not very good at scripting language at all, but could the problem be fixed by just adding 
 -classpath %CLASSPATH% in the final command line? 

It seems that we don't need to introduce DEPENDENCIES variable.